package.json

```diff
@@ -16,43 +16,43 @@
     "intl": "1.2.5",
     "jail-monkey": "0.2.0",
     "mattermost-redux": "mattermost/mattermost-redux",
-    "prop-types": "15.6.0",
-    "react": "16.2.0",
+    "prop-types": "15.6.1",
+    "react": "^16.3.0-alpha.1",
```

Comment:
I trust that you have valid reason in stepping up early on at 16.3 even still on alpha.
I believe there's so much hype in 16.3, and I'm excited to implement some of those in the near future.

Reply:
Reason being that the react native upgrade script did it automatically as is a direct dependency